perceval/backends/github.py

```diff
@@ -187,6 +232,22 @@ def get_issues(self, start=None):
                 logger.debug("Rate limit: %s" %
                      (r.headers['X-RateLimit-Remaining']))
 
+    def get_user(self, login):
+        user = None
+
+        if login in self._users:
+            user = self._users[login]
```

Comment:
I don't like in general to return inside if/else for/while logic because it is hard to follow the logic, but in this simple case, ok, I will do that.